src/common/containers/landing.js

```diff
@@ -43,31 +43,54 @@ class Landing extends Component {
 
         <section className={styles.section}>
           <div className={ `${styles.row} ${containerStyles.wrapper}` }>
-            <ListDividedState className={ styles.rowItem }>
+            <ListDividedState className={ styles.rowItemGrow }>
               <li>Scale to millions of installs</li>
               <li>Available on all clouds and Linux OSes</li>
               <li>No need for build infrastructure</li>
             </ListDividedState>
 
-            <ListDividedState className={ styles.rowItem }>
+            <ListDividedState className={ styles.rowItemGrow }>
               <li>Automatic updates for everyone</li>
               <li>Roll back versions effortlessly</li>
               <li>FREE for open source projects</li>
             </ListDividedState>
           </div>
         </section>
 
-        <div className={styles.container}>
-          {/* TODO brand logos */}
-          {/* TODO workflow */}
+        <section className={styles.section}>
+          <div className={ `${styles.row} ${containerStyles.wrapper}` }>
+            {/* TODO put in actual logos */}
+            <img className={ styles.rowItem } src='http://placehold.it/180x130?text=BRAND+LOGO' />
+            <img className={ styles.rowItem } src='http://placehold.it/180x130?text=BRAND+LOGO' />
+            <img className={ styles.rowItem } src='http://placehold.it/180x130?text=BRAND+LOGO' />
+            <img className={ styles.rowItem } src='http://placehold.it/180x130?text=BRAND+LOGO' />
+          </div>
+        </section>
 
-          <div>
-            <p>Snaps are secure, sandboxed, containerised applications, packaged with their dependencies for predictable behavior. With the Snap Store, people can safely install apps from any vendor on mission-critical devices and PCs.</p>
-            <Anchor href="http://snapcraft.io">More about snaps</Anchor>
+        <section className={styles.section}>
+          <div className={containerStyles.wrapper}>
+            {/* TODO put in actual workflow image */}
+            <img className={ styles.workflowImage} src='http://placehold.it/920x480?text=WORKFLOW' />
+            <div className={ styles.centeredButton }>
+              <Anchor  href="/auth/authenticate">Get Started Now</Anchor>
+            </div>
           </div>
+        </section>
 
-          {/* TODO testimonials */}
-        </div>
+        <section className={styles.section}>
+          <div className={ `${styles.row} ${containerStyles.wrapper}` }>
+
+
+            <div className={styles.rowItemTwoThirds}>
+              <p className={styles.snaps}>Snaps are secure, sandboxed, containerised applications, packaged with their dependencies for predictable behavior. With the Snap Store, people can safely install apps from any vendor on mission-critical devices and PCs.</p>
+              <Anchor href="http://snapcraft.io">More about snaps</Anchor>
```

Comment:
This should be `https` from the start, since the `http` version redirects to `https` anyway.